refinery/templates/core/about.html

```diff
@@ -1,6 +1,6 @@
 {% extends "base.html" %}
 {% load markup %}
-{% load flatblock_tags %}
```

Comment:
Are the `flatblocks` and `registration` apps intertwined somehow? Or are these just changes due to the `flatblocks` upgrade in `jkmarx/django-upgrade-1.8`

Reply:
They are not intertwined. I just didn't make another pull request for flatblocks. The changes are due to the apps upgrade.